.github/workflows/pr-linter.yml

```diff
@@ -0,0 +1,23 @@
+# https://octokit.github.io/rest.js
+# https://github.com/actions/toolkit/blob/master/packages/github/src/context.ts
+
+name: PR Linter
+on: pull_request
+
+jobs:
+  mandatory-changes:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install packages
+        run: cd .github/actions/prlinter && npm install
```

Comment:
If you want to honor your `package-lock.json`, you need to `npm ci` instead of `npm install`.

Reply:
Ha! good to know, thanks